packages/apps/reputation-oracle/server/src/main.ts

```diff
@@ -8,16 +8,20 @@ import { AppModule } from './app.module';
 import { useContainer } from 'class-validator';
 import { ServerConfigService } from './config/server-config.service';
 import logger, { nestLoggerOverride } from './logger';
+import { getBodyParserOptions } from '@nestjs/platform-express/adapters/utils/get-body-parser-options.util';
```

Comment:
We should avoid importing anything by direct paths from libraries because it violates their encapsulation and can break at any moment, instead we can have our own implementation